Gamenights/NewGameResultViewController.swift

```diff
@@ -15,6 +15,11 @@ class NewGameResultViewController: UIViewController {
     @IBOutlet weak var notesTextView: UITextView!
 
     var groupId: String?
```

Comment:
since you have a `group` member now, how about getting rid of the `groupId` field so there's a single source of truth here?